pkg/json2table/json2table_unit_test.go

```diff
@@ -0,0 +1,128 @@
+//go:build unit
+// +build unit
```

Comment:
we don't need annotations for unit tests, only for integration tests.

Also, the file name can be just `json2table_test.go`

However I have no problem with keeping these if you think they are useful